support-frontend/assets/pages/contributions-landing/components/PaymentMethodSelector.tsx

```diff
@@ -112,231 +104,277 @@ const mapDispatchToProps = {
 									/>
-									<div
-										css={css`
-											font-size: small;
-											font-style: italic;
-											margin-left: 40px;
-											padding-bottom: 6px;
-											color: #767676;
-											padding-right: 40px;
-										`}
+								}
+							/>
+						))}
+
+						{contributionTypeIsRecurring(props.contributionType) &&
+							props.existingPaymentMethods &&
+							props.existingPaymentMethods.length > 0 &&
+							fullExistingPaymentMethods.length === 0 && (
+								<li className="form__radio-group-item">
+									...or{' '}
+									<a
+										className="reauthenticate-link"
+										href={getReauthenticateUrl()}
 									>
-										Used for your{' '}
-										{subscriptionsToExplainerList(
-											existingPaymentMethod.subscriptions.map(
-												subscriptionToExplainerPart,
-											),
-										)}
-									</div>
-								</>
-							),
-						)}
-					{paymentMethods.map((paymentMethod) => (
-						<Radio
-							id={`paymentMethodSelector-${paymentMethod}`}
-							name="paymentMethodSelector"
-							type="radio"
-							value={paymentMethod}
-							onChange={() => {
-								onPaymentMethodUpdate(paymentMethod, props);
-							}}
-							checked={props.paymentMethod === paymentMethod}
-							label={renderLabelAndLogo(paymentMethod)}
-							cssOverrides={radioCss}
-						/>
-					))}
-					{contributionTypeIsRecurring(props.contributionType) &&
-						props.existingPaymentMethods &&
-						props.existingPaymentMethods.length > 0 &&
-						fullExistingPaymentMethods.length === 0 && (
-							<li className="form__radio-group-item">
-								...or{' '}
-								<a
-									className="reauthenticate-link"
-									href={getReauthenticateUrl()}
-								>
-									re-enter your password
-								</a>{' '}
-								to use one of your existing payment methods.
-							</li>
-						)}
+										re-enter your password
+									</a>{' '}
+									to use one of your existing payment methods.
+								</li>
+							)}
+					</>
 				</RadioGroup>
 			) : (
-				noPaymentMethodsErrorMessage
+				<GeneralErrorMessage
+					classModifiers={['no-valid-payments']}
+					errorHeading="Payment methods are unavailable"
+					errorReason="all_payment_methods_unavailable"
+				/>
 			)}
 		</div>
 	);
 }
 
+// ----- Helper components ----- //
+
+function Legend() {
+	return (
+		<div className="secure-transaction">
+			<legend id="payment_method">
+				<ContributionChoicesHeader>Payment Method</ContributionChoicesHeader>
+			</legend>
+			<SecureTransactionIndicator modifierClasses={['middle']} />
+		</div>
+	);
+}
+
+interface PaymentMethodLabelProps {
+	label: string;
+	logo: JSX.Element;
+	isChecked: boolean;
+}
+
+function PaymentMethodLabel({
+	label,
+	logo,
+	isChecked,
+}: PaymentMethodLabelProps) {
+	return (
+		<div css={styles.labelContainer} data-checked={isChecked.toString()}>
+			<div>{label}</div>
+			{logo}
+		</div>
+	);
+}
```

Comment:
I factored this out of the `renderLabelAndLogo` and `renderExistingLabelAndLogo`. I've also moved the styles directly on to this component. Before they were being passed as the `cssOverrides` prop to the `Radio` component, but that required some knoweldge of the internals to target this div - now we don't need to do that!